### PR TITLE
Nc games 3 get api categories

### PR DESCRIPTION
**Pull request by JLaw96**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          .expect(200)
+          .then(({ body }) => {
+            const { categories } = body;
+            categories.forEach((category) => {
+              expect(category).toMatchObject({
+                slug: expect.any(String),
+                description: expect.any(String),
+              })
+            })
+          });
```
Comment (pippaamy) [inline __tests__/app.test.js:52]: Need a 404 test to test for /api/pathnotfound :)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      });
+    });
+    describe("GET /api/categories", () => {
+      it("200: GET - should respond with an array containing the categories object with the correct data types for the keys values.", () => {
+        return request(app)
+          .get("/api/categories")
+          .expect(200)
+          .then(({ body }) => {
+            const { categories } = body;
+            categories.forEach((category) => {
```
Comment (pippaamy) [inline __tests__/app.test.js:46]: Need to check length of array before doing the forEach

**Review by pippaamy — CHANGES_REQUESTED** [review]
Good work! A couple of changes to make - then pop in another pull request :)

**Review by Keb84 — APPROVED** [review]
Looks good Johnny. Feel free to merge this now 👍

**Pull request merged**